<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setrumsDBDialog</class>
 <widget class="QDialog" name="setrumsDBDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mzrollDB Loading Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="3">
    <widget class="QGroupBox" name="grpMzkitchenOptions">
     <property name="title">
      <string>mzkitchen options</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout">
      <item row="2" column="1">
       <widget class="QPushButton" name="btnSelectLibrary">
        <property name="text">
         <string>Launch Library Dialog</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="cmbLoadedLibraries">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblAddLibrary">
        <property name="text">
         <string>Add rumsDB library:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblSelectLibrary">
        <property name="text">
         <string>rumsDB library:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="lblRequireMzkitchen">
        <property name="toolTip">
         <string>If the library was not generated by mzkitchen, the association will not work</string>
        </property>
        <property name="text">
         <string>If this mzrollDB was generated by mzkitchen, select the msp library file generated by that mzkitchen run.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="btnUseSelected">
     <property name="text">
      <string>Use Selected Library</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="btnCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="btnNoLibrary">
     <property name="text">
      <string>Proceed without library</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="chkmzrollDBLoadLibs">
     <property name="text">
      <string>Automatically load libraries referenced in mzrollDB file</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
